client/view.js

```diff
@@ -73,18 +68,30 @@ export default class View
 
 	#addPlayer(hand)
 	{
+		const $container = document
 		var $player = this.$playerRowTemplate.cloneNode(true);
 		$player.querySelector('.js-id').textContent = hand.playerId;
 		this.#playerRows[hand.playerId] = $player;
 		$player.querySelector('.js-name').textContent = hand.playerName;
-		$player.querySelector('.js-owner').textContent = hand.gameOwner;
+		$player.querySelector('.js-owner').innerHTML = hand.gameOwnerButton;

```

Comment:
see comment below. Just hide or show the HTML instead of getting the HTML from the js